### ISSUE TITLE
Nan errors in calculating "GroundTruth"

### ISSUE THREAD

**Issue by hhssv** [justkolesov/FieldMatching#2]
Hi.

I found the error when calculating electric fields in `src/efm_field`.

In the "GroundTruth" function, it omits an epsilon to avoid numerical error in a division operation.

`coeff_x = distance_x / (torch.sum(distance_x, dim=1, keepdim=True)  ) # [b,B,1]`
`coeff_y = distance_y / (torch.sum(distance_y, dim=1, keepdim=True)  ) # [b,B,1]`

It differs from PFGM as you mentioned in the paper.

Is there any reason to omit the epsilon?

Also, I can not reproduce your CIFAR-10 results provided in the paper.

Could you provide correct settings for training CIFAR-10?

Thanks.
